lib/logflare_web/controllers/plugs/verify_resource_ownership.ex

```diff
@@ -11,23 +11,24 @@ defmodule LogflareWeb.Plugs.VerifyResourceOwnership do
   alias LogflareWeb.Api.FallbackController
   def init(_opts), do: nil
 
-  # no user set. (handles endpoints with no auth)
-  def call(%{assigns: assigns} = conn, _opts) when is_map_key(assigns, :user) == false, do: conn
+  def call(%{assigns: %{endpoint: %Query{enable_auth: false}}} = conn, _opts) do
+    conn
```

Comment:
we should rethink this at a later stage, having the fallback controller handle scenarios like this will be a tricky thing to debug at a later stage.

Fallback controllers should only handle errors since even their catch all behavior is already risky enough 😓 

But it can be a future us problem

Reply:
something looks off with the module and the file name, i'll check this. This logic isn't for fallback controller

Reply:
This isn't the fallback controller, it is a separate plug that is checking for ownership. We do a passthrough if we don't need to check., which in thie case is when auth is disabled.

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️🤦‍♂️ my bad that was stupid..